<?xml version="1.0" encoding="utf-8"?>
<launch>
    <node name="rviz" pkg="rviz" type="rviz" output="screen" args="-d $(find am_traj)/config/traj.rviz"/>

    <node pkg="rnw_ros" type="test_traj_node" name="test_traj_node"/>

    <node pkg="rnw_ros" type="visualize_traj_node" name="traj_visualization" output="screen" required="true">
        <remap from="~poly_traj" to="/poly_traj_test"/>
        <param name="lift_dt" type="double" value="0.05"/>
        <param name="length_g" type="double" value="0.5"/>
    </node>

</launch>
